contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3702,6 +3702,31 @@ END;
 $BODY$
 LANGUAGE plpgsql IMMUTABLE PARALLEL SAFE;
 
+-- wrapper functions for translate --
+CREATE OR REPLACE FUNCTION sys.translate(string sys.VARCHAR, characters sys.VARCHAR, translations sys.VARCHAR)
+RETURNS sys.VARCHAR
+AS $$
+BEGIN
+    IF length(characters) != length(translations) THEN
```

Comment:
nit: do we need to specify name space here?

Reply:
I don't think we need to specify that explicitly, since we haven't defined `length` in `sys` namespace so it is picking up `pg_catalog.length` by default